### DISCUSSION TITLE
Working example of MemPooling operation?

### DISCUSSION THREAD

**Discussion by mohit-raghavendra** [pyg-team/pytorch_geometric#2623] [Q&A]
I have been using PyTorch Geometric extensively over the past year and I have found it to be one of the most comprehensive Graph ML libraries! Kudos to the developers.

I was wondering if there is a working example for the **MemPooling** technique introduced in the paper _Memory-Based Graph Networks_ in ICLR 2020. I was attempting to club it with other models in the library but I keep running into issues related to detach() during backprop. 

Is there an example usage of **MemPooling** similar to the other techniques in the library? If so, a link to it would be highly appreciated! 

Thank you!

**Comment by rusty1s** [comment]
We do not have an example yet, but please feel free to post your error and a minimal example to reproduce. I'm sure we can identify what's going wrong :)

**Comment by wsad1** [comment]
About the working example.
I'll try to add one in the coming weeks. In the meanwhile you could check [this code](https://github.com/amirkhas/GraphMemoryNet/blob/master/train.py) from the authors of the paper. it uses their own implementation of the Memory based GNN, but it shows the variable learning rate schedule for different parameters, as described in the paper.

**Comment by mohit-raghavendra** [comment]
Thank you for your replies @rusty1s and @wsad1. I will look into the original implementation and the example that @wsad1 is going to add. If the issue still persists, I will post my code to reproduce the error. Thanks!
